### ISSUE TITLE
[Bug] 通过xAxis.axisTick.interval控制刻度展示和实际展示效果不一致

### ISSUE THREAD

**Issue by sunjiesama** [apache/echarts#20424]
### Version

5.5.0

### Link to Minimal Reproduction

https://echarts.apache.org/examples/zh/editor.html?c=line-smooth&code=PYBwLglsB2AEC8sDeAoWsAeBBDEDOAXLANprrJnmxgCeIApkQOQDGAhmPQObABONTADSVyAEw5sixJgFkYQ2EwAqAV3oKmAdXqiNSgBYqmAXREBfYVVRV0tBs3ace_ISPTiwkkkz1qN_40sbEGA8CEgYZgAjYDAwYABbVxtYNlw8ABk2KPoAGyJrFNgAMz4Ejk5eZgAdaFqkADc2XLUzJjdYCw60_CUIFgBrAo70PH1gAHciMF41IJSIaEqm_NgACgB9CEFYBoBKBAA-CiLyFhg8YFz6ADpc4C41hp2tnYaEeERlPz2AbhHyLx6GAVLw4O9Pl9VOp_qcuqdrtAuGB9EQAMwAVg6ZnMZECZBoOHwwyodkYihWfjI8NgeHovAg9EIJBEhSoHi8xAAHAAmAAMOwAnGieUK-QBGIVogAs-JSZOYuUW6nm5DwCWAsVR1Fm9Fx6FMZl-QA

### Steps to Reproduce

详情请参考最小复现

### Current Behavior

第二条刻度线的第一个刻度应该从第二个点开始展示

### Expected Behavior

第二条刻度线的第一个刻度从第一个就开始展示了

### Environment

```markdown
- OS:
- Browser:
- Framework:
```

**Comment by helgasoft** [comment]
yes, looks like a bug somewhere in [makeLabelsByNumericCategoryInterval](https://github.com/apache/echarts/blob/845b6a32c2af35458d11efcba22a6820fb6aeb41/src/coord/axisTickLabelBuilder.ts#L377).
For a (weird) workaround use ```return (_i==0 || v=='Tue');```.

**Comment by sunjiesama** [comment]
> yes, looks like a bug somewhere in [makeLabelsByNumericCategoryInterval](https://github.com/apache/echarts/blob/845b6a32c2af35458d11efcba22a6820fb6aeb41/src/coord/axisTickLabelBuilder.ts#L377). For a (weird) workaround use `return (_i==0 || v=='Tue');`.

Thank you for your answer. This method has indeed achieved what I wanted, but the first scale is still displayed. I'm thinking about it

**Comment by helgasoft** [comment]
> but the first scale is still displayed

it comes from ```_i==0``` 😄  So you either have Mon + Tue ticks, or only Mon...  I said it's weird

**Comment by sz-p** [comment]
Look like a BUG in [fixOnBandTicksCoords](https://github.com/apache/echarts/blob/master/src/coord/Axis.ts#L303-L306) 

Try return 2 tick it will be work

See :

https://echarts.apache.org/examples/zh/editor.html?c=line-smooth&code=PYBwLglsB2AEC8sDeAoWsAeBBDEDOAXLANprrJnmxgCeIApkQOQDGAhmPQObABONTADSVyAEw5sixJgFkYQ2EwAqAV3oKmAdXqiNSgBYqmAXREBfYVVRV0tBs3ace_ISPTiwkkkz1qN23UFFE0sbEGA8CEgYZgAjYDAwYABbVxtYYAAzTLx6MCIARgAGUKo2XDwAGQhoRgp09Dx9YAB3Iky2ABtct1gLXvL8SrZY-k6iawbMvmSOTl5mAB1oZaQANy61MyZe_vTBvCUIFgBrCd7G5rbqXjVSmxr5jfHYAAoAfQggtYBKBAA-eoNdAsGB4YCdegAOk6wC4rzWQU-3wQ8EQyj8PwA3BdyLw8ipeHA1qj0ap1LAAD6U2AktHogJMWA44F7BqQ6BcMD6IgAZgArLtzGRjKEaDh8OcqHY6kxnn4yGzcrwIPRCCQRJMqB4vMQABwAJhKsAAnLyDUETUUCpbeQAWUW9GXMTo1dT3RrJYAJHk3NTC9CmMxYoA
